docs/operator-manual/cluster-bootstrapping.md

```diff
@@ -119,4 +119,29 @@ metadata:
   - resources-finalizer.argocd.argoproj.io
 spec:
  ...
-``` 
+```
+
+### Ignoring some differences in managed applications
```

Comment:
Seems like this would make more sense in docs/user-guide/diffing.md. Perhaps in an example section that could contain known patterns.

What do you think?

Reply:
I'd like to keep some reference here, since the page is dedicated to app of apps and that's where people would come to when searching for app of apps. They'd probably not go to diffing page right away. The filters are a bit specific for app of apps, so it seems to make more sense to keep them here. Though I can potentially add changes to the diffing page and link them from this page.

Reply:
@agaudreault if you Google "Argo CD app of apps" this page is the first result and is essentially the canonical source for app of apps. I think we should leave changing that pattern to another PR. In the meantime, for this PR @andrii-korotkov-verkada lets just add a link from diffing customization page to this one to discuss app of apps.